src/sentry/static/sentry/app/views/releases/list/releaseLanding.tsx

```diff
@@ -122,7 +122,7 @@ class Promo extends React.Component<PromoProps> {
       eventKey: 'releases.landing_card_viewed',
       eventName: 'Releases: Landing Card Viewed',
       organization_id: parseInt(organization.id, 10),
-      project_id: parseInt(project.id, 10),
+      project_id: project && parseInt(project.id, 10),
```

Comment:
Could we change the type to be optional and callout that the HoC is wrong in this case.

Reply:
I can do that in a followup PR, this is a hotfix.

Reply:
I think this is good for a short term fix. Updating the types of withProject and all its consumers is a good follow up.

Reply:
yep